daemon/hostdaemon.go

```diff
@@ -161,7 +174,7 @@ func (d *HostDaemon) Start() {
 		return d.delHandler(r)
 	}
 
-	d.cniserver = cniserver.NewCNIServer(add, del)
+	d.cniserver = cniserver.NewCNIServer(add, del, cniserver.WithSocketPath(d.cniServerPath))
```

Comment:
I think this code change should be squash with the first commit in this series for "each commit" should compile.